packages/dashmate/configs/migrations.js

```diff
@@ -364,4 +364,15 @@ module.exports = {
 
     return configFile;
   },
+  '0.23.0-alpha.2': (configFile) => {
```

Comment:
You can use current version so migrations will be applied for developers who will work with dev branch as well

Reply:
Making the same versions makes me LGTM fire a warning